torch/_export/non_strict_utils.py

```diff
@@ -125,11 +125,12 @@ def make_fake_inputs(
     #   - output_graph.py fakifies inputs.
     #   - [post-tracing] guards.py processes input shape equalities.
 
-    combined_args = _combine_args(
-        nn_module, args, kwargs, _is_torch_jit_trace=_is_torch_jit_trace
-    )
+    combined_args = _combine_args(nn_module, args, kwargs)
```

Comment:
Why do we not have _is_torch_jit_trace flag anymore? This was needed when we retrace torch.jit.trace-d modules to export.

Reply:
Oh, for this callsite it's not needed, we just need the dictionary binding